src/presentation/test/integration/registration/gateway/check-your-answers.test.ts

```diff
@@ -0,0 +1,60 @@
+import request from "supertest";
+import { createApiClient } from "@companieshouse/api-sdk-node";
+
+import appRealDependencies from "../../../../../app";
+import {
+  APPLICATION_SUBMITTED_URL,
+  CHECK_YOUR_ANSWERS_URL
+} from "../../../../controller/registration/url";
+import RegistrationPageType from "../../../../controller/registration/PageType";
+import sdkMock, {
+  putTransaction
+} from "../../mock/sdkMock";
+import { getUrl } from "../../../utils";
+import enTranslationText from "../../../../../../locales/en/translations.json";
+
+jest.mock("@companieshouse/api-sdk-node");
+
+const mockCreateApiClient = createApiClient as jest.Mock;
+mockCreateApiClient.mockReturnValue(sdkMock);
+
+jest.mock("../../../../../infrastructure/repository/CacheRepository");
+
+describe("Gateway Transaction", () => {
```

Comment:
description should have something to indicate it's for the check you answers page

Reply:
Done.